doc/sql/user_message.sql

```diff
@@ -0,0 +1,43 @@
+/*
+SQLyog 企业版 - MySQL GUI v8.14 
+MySQL - 5.7.21-log : Database - user_message
+*********************************************************************
+*/

+
+/*!40101 SET NAMES utf8 */;
+
+/*!40101 SET SQL_MODE=''*/;
+
+/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+CREATE DATABASE /*!32312 IF NOT EXISTS*/`user_message` /*!40100 DEFAULT CHARACTER SET utf8 */;

```

Comment:
create database if not exists database_name default character set UTF8MB3 collate utf8_general_ci;